bedrock/firefox/templates/firefox/os/_in_the_news.html

```diff
@@ -1,16 +0,0 @@
-{% set feed_links = firefox_os_feed_links(LANG) %}
```

Comment:
Actually, you may want to leave doing this for now - it also means removing a cron job, which could be a bit fiddly. I can file a follow up bug to do this if you don't want to go down that rabbit hole.